src/ExpressionLanguage/ExpressionFunction/GraphQL/IsTypeOf.php

```diff
@@ -8,12 +8,15 @@
 
 final class IsTypeOf extends ExpressionFunction
 {
-    public function __construct($name = 'isTypeOf')
+    public function __construct()
     {
         parent::__construct(
-            $name,
+            'isTypeOf',
             function ($className) {
                 return \sprintf('(($className = %s) && $value instanceof $className)', $className);
+            },
+            function ($arguments, $className): bool {
+                return $className && $arguments['prevValue'] instanceof $className;
```

Comment:
`prevValue` ?

Reply:
As you know there are predefined expression variables used in resolve context: `args`, `context`, `info` and `value`.

The [Expression](https://symfony.com/doc/current/reference/constraints/Expression.html) validation constraint also has a predefined variable called `value`.

The thing is, that you can use all variables from the resolve context in the expression context, so the variable names conflict. Thats why, when you want to access the variable `value` from the resolve context in your expression, you need to use `prevValue` instead, because the Expression constraint has it's own `value`.

Please reference the [documentation](https://github.com/murtukov/GraphQLBundle/blob/feature/validation_v.0.13/docs/validation/index.md#using-built-in-expression-functions).